test/test-rule-help-version.js

```diff
@@ -0,0 +1,16 @@
+var https = require('https');
+var path = require('path');
+var assert = require('assert');
+var packageJSON = require(path.join(__dirname, '../package.json'));
+
+var version = packageJSON.version.substr(
```

Comment:
This looks a little more complicated and error-prone than necessary. What about:

```js
var [major,minor] = packageJSON.version.split('.')
var url = `https://dequeuniversity.com/rules/axe/${major}.${minor}`
// ...
```

Reply:
Also, since this test is run **in a Node.js context**, we can use "modern" JS syntax. We don't have PhantomJS getting in the way here.

Reply:
Except that we our jslint blocks all es6 syntax in the `test` dir.

Reply:
Bummer OK